src/routes/profile.ts

```diff
@@ -476,20 +477,20 @@ router.get('/:profileUsername/edit', checkProfileOwnership, (req, res) => {
           .then((patreonIdObj) => {
             res.render('profile/edit', {
               userData: foundUser,
-              patreonLoginUrl: loginUrl,
+              patreonLoginUrl: patreonAgent.getPatreonAuthorizationUrl(),
```

Comment:
Let's make a new folder at the root called `clients` and move `discord`, `s3` and `patreon` stuff into there :). 

Then, inside the patreon folder, add a new file called `constants.ts` with the patreon login URL. 
Then, the routes import it for use here, and the controller also imports `constants.ts`.

Reply:
By moving the urls to another folder constants, still need to construct URL in patreonController since need to pass in query params (clientId, state etc)